doc/2018/2018-08-07.md

```diff
@@ -0,0 +1,84 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## cloud-init
+
+  - docs: Fix example cloud-init analyze command to match output.
+    [Wesley Gao]
+  - netplan: Correctly render macaddress on a bonds and bridges when
+    provided. (LP: #1784699)
+  - tools: Add 'net-convert' subcommand command to 'cloud-init devel'.
+  - redhat: remove ssh keys on new instance. (LP: #1781094)
+  - Use typeset or local in profile.d scripts. (LP: #1784713)
+  - OpenNebula: Fix null gateway6 [Akihiko Ota] (LP: #1768547)
+  - Canonical's Chad Smith is granted [PPU rights (per package upload)](https://wiki.ubuntu.com/UbuntuDevelopers#PerPackage) into Ubuntu for cloud-init
+    and curtin packages.
```

Comment:
These are all indented? Is there are a reason why? Ideally it should be no extra space.

Reply:
fixed